test/dlc_tests/eks/eks_manifest_templates/kubeflow/kfctl_aws.yaml

```diff
@@ -116,5 +116,5 @@ spec:
       #- eksctl-kubeflow-aws-nodegroup-ng-a2-NodeInstanceRole-xxxxxxx
   repos:
   - name: manifests
-    uri: https://github.com/kubeflow/manifests/archive/v1.0.2.tar.gz
```

Comment:
I dont think this is the only change, can you do a diff between the latest available and this file? I did remove some components that we do not need but we can compare other items

Reply:
This is the first time a change is made to this file since inception. Are you referring to a different file for comparison?